onprc_ehr/resources/schemas/dbscripts/sqlserver/onprc_ehr-12.395-12.396.sql

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
Since we're already at 18.10 in the 20.7 branch we'll have to merge these scripts manually.

Reply:
onprc_ehr and onprc_billing scripts are now rolled up:
onprc_ehr upgrade scripts (12.20 to 17.705) consolidated to onprc_ehr-0.00-18.10.
onprc_billing scripts (12.30 to 17.502) consolidated to onprc_billing-0.00-18.10.

Reply:
Sorry I wasn't more specific. I wasn't worried about rolling up the scripts. I'm worried that devs on the 20.7 branch will already have the onprcEhrModules checked out, and they'll be on version 18.10 of the schema. Therefore, they'll never run these scripts, whether they're incrementals or rollups.